.github/workflows/ci-sharktank-and-shortfin.yml

```diff
@@ -70,31 +70,5 @@ jobs:
             iree-runtime \
             "numpy<2.0"
 
-      - name: Run llama test
-        run: ./build_tools/integration_tests/llama_export_compile_serve.sh
```

Comment:
A little curious why we put this into build_tools instead of its own tests dir

If it's only because `llama_export_compile_serve.sh` is in there, we should probably make its own tests dir

I i know the org has the convention of putting temporary build / test / run scripts in `build_tools` and i don't think this is an obvious and intuitive place to be finding `integration_tests` in

It would make more sense to put this into a top level `tests` or `integration_tests` dir

ignore the above if you've received feedback from other people that caused you to delete the top level tests directory

Reply:
Thread discussing moving the test to `build_tools/integrations_tests`: https://github.com/nod-ai/SHARK-Platform/pull/373#discussion_r1823458520